src/OSPSuite.Presentation/Views/Charts/IDataBrowserView.cs

```diff
@@ -16,5 +17,16 @@ public interface IDataBrowserView : IView<IDataBrowserPresenter>, IViewWithColum
       /// Returns all selected <see cref="DataColumnDTO"/> and their descendants.
       /// </summary>
       IReadOnlyList<DataColumnDTO> SelectedDescendantColumns { get; }
+
+      /// <summary>
+      /// sets the group row format of the gridView to the specified string.
+      /// </summary>
+      void SetGroupRowFormat(string format);
+
+      /// <summary>
+      ///    Dictionary that specifies the Group Values (Dictionary Key) in the DataBrowser Grid for which the group title should be
```

Comment:
So,ething is off here with the comments

Reply:
If feels that this is going to be always the one. Do we need to have this?
In fact, it feels like we want to rename the columns all the time? Why don't we do this at binding time ?

Reply:
Yes this is actually always called (I was under the impression the event was triggered once, when the grouping was happening). I have used this event according to the devexpress documentation. We are not really renaming the column name, but the value of the "Data Origin" column when the grouping is happening: "CalculationAuxiliary" -> "Simulation". 

I have thought a lot about a possible workaround to this, but as far as I understand the only other possible solution is to handle the gridView.EndGrouping event that does actually get triggered at the end of the grouping. The problem here (as far as I understand always ) is that traversing the grouping rows is not that straightforward. Again, let's discuss this in the meeting